src/Renderers/ErrorPageRenderer.php

```diff
@@ -15,6 +15,21 @@ class ErrorPageRenderer
 {
     public function render(Throwable $throwable): void
     {
+        $vitejsAutoRefresh = '';
+
+        if (class_exists('Illuminate\Foundation\Vite')) {
+
+            $vite = app(\Illuminate\Foundation\Vite::class);
+
+            $hotFile = version_compare(app()->version(), '9.27.0') >= 0
```

Comment:
Reason for `version_compare` instead of `method_exists`:
Because the method already existed before Laravel `v9.27.0`, that call would also return `true`.
`Vite::hotFile()` is only public after the release of `v9.27.0` and therefor I used a version compare.

Reply:
If you want, it would also be possible to increase the required `illumunate/support` version to `^9.27`

Reply:
Yeah, let's do that! 👍